hibernate-micrometer/hibernate-micrometer.gradle

```diff
@@ -2,13 +2,20 @@ description = 'Integration for Micrometer metrics into Hibernate as a metrics co
 
 apply from: rootProject.file( 'gradle/published-java-module.gradle' )
 
+configurations {
```

Comment:
These Configurations are transitive by default (see the Gradle forum post).  This is pointless.

Reply:
Whooops misunderstood 😄